pcsx2/gui/AppMain.cpp

```diff
@@ -547,18 +570,35 @@ void Pcsx2App::LogicalVsync()
 	FpsManager.DoFrame();
 	
 	if (EmuConfig.Gamefixes.FMVinSoftwareHack) {
-		if (EnableFMV == 1) {
+		if (EnableFMV) {
 			Console.Warning("FMV on");
 			DoFmvSwitch();
-			EnableFMV = 0;
+			EnableFMV = false;
 		}
 
 		if (FMVstarted){
 			int diff = cpuRegs.cycle - eecount_on_last_vdec;
 			if (diff > 60000000 ) {
```

Comment:
What kind of voodoo is that actually?
Is this an empirical number?
Would be interesting to know whether the sw-fmv switch would still lead to random crashing if this number would be different.